docs/how-to/jupyter-notebook/create-notebook.rst

```diff
@@ -16,42 +16,52 @@ Before creating a notebook, ensure you have the following:
 - DSS CLI installed on your workstation
 - DSS initialized
 
-Creating a Notebook
--------------------
-
-1. **Select an image**:
+Select an Image
+---------------
 
-    Before creating a notebook, you need to select an image that includes the packages and toolkits you need.  To see a list of recommended images and their aliases, see:
+Before creating a notebook, you need to select an image that includes the packages and toolkits you need.
+To see a list of recommended images and their aliases, see:
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        dss create --help
+    dss create --help
 
-    The help text includes a list of recommended images and aliases so you don't need to type the full image name.  For this guide, we will use the image `kubeflownotebookswg/jupyter-scipy:v1.8.0`
+The help text includes a list of recommended images and aliases so you don't need to type the full image name.
+For this guide, we will use the image `kubeflownotebookswg/jupyter-scipy:v1.8.0`
 
-2. **Create the notebook**:
+Launch the Notebook
+-------------------
 
-    Create a new notebook using ``dss create``:
+Create a new notebook using ``dss create``:
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        dss create my-notebook --image kubeflownotebookswg/jupyter-scipy:v1.8.0
+    dss create my-notebook --image kubeflownotebookswg/jupyter-scipy:v1.8.0
 
-    This will pull the notebook image and start a Notebook server, printing the URL of the notebook once complete.  Expected output:
+This will pull the notebook image and start a Notebook server, printing the URL of the notebook once complete.  Expected output:
 
-    .. code-block:: none
+.. code-block:: none
 
-        [INFO] Executing create command
-        [INFO] Waiting for deployment test-notebook in namespace dss to be ready...
-        [INFO] Deployment test-notebook in namespace dss is ready
-        [INFO] Success: Notebook test-notebook created successfully.
-        [INFO] Access the notebook at http://10.152.183.42:80.
+    [INFO] Executing create command
+    [INFO] Waiting for deployment test-notebook in namespace dss to be ready...
+    [INFO] Deployment test-notebook in namespace dss is ready
+    [INFO] Success: Notebook test-notebook created successfully.
+    [INFO] Access the notebook at http://10.152.183.42:80.
 
-3. **Access the notebook**:
+Access the notebook
+-------------------
 
-    To :doc:`access the Notebook </how-to/jupyter-notebook/access-ui>`, use the URL provided in the output.
+To :doc:`access the Notebook </how-to/jupyter-notebook/access-ui>`, use the URL provided in the output.
 
 Conclusion
```

Comment:
I am not sure these conclusions sections are actually needed for how-to guides. I will add this to the backlog list.